string action_type
string parameters
---
bool execution_successful
string result_message
